Tests/configure_and_test_integration_instances.py

```diff
@@ -1022,10 +1022,13 @@ def nightly_install_packs(build, threads_print_manager, install_method=install_a
 
 def install_nightly_pack(build, prints_manager):
     threads_print_manager = ParallelPrintsManager(len(build.servers))
-    nightly_install_packs(build, threads_print_manager, install_method=install_all_content_packs)
+    if build.is_nightly:
```

Comment:
why do we need this if?

Reply:
If it's a private run, we don't need to install all the packs. We install the necessary ones at a later stage. It's meant to speed this part of the build up ever so slightly.